docs/config.md

```diff
@@ -17,6 +17,12 @@ The template of the SVG to respond with. Must take `count`, `width`, `recWidth`,
 ## `ENABLE_SSL`
 Enable SSL. Defaults to false for debugging purposes. Can be overridden with the `ENABLE_SSL` environment variable.
 
+## `EXPOSE_METRICS`
+Enable [Prometheus](https://prometheus.io) metrics at the `/metrics` endpoint. Defaults to false for performance reasons. Can be overridden with the `EXPOSE_METRICS` environment variable. However, use this feature with care. It is recommended only for moderately large deployments of _hit-counter_, as a separate metric will be exposed for every URL in your data base.  
```

Comment:
> It is recommended only for moderately large deployments of _hit-counter_, as a separate metric will be exposed for every URL in your data base.  

Do you mean it's recommended for instances with low amounts of URLs stored?

Reply:
Exactly. Do you have doubts about it? Also, feel free to clarify that statement if you think it might be confusing.

Reply:
Sorry, I read

> It is recommended only for moderately large deployments of hit-counter

as saying it's only recommended to use this for instances with a large amount of data. I now see you are saying the instance needs to be large to hold all the URLs. My bad on this one.